ui/components/app/wallet-overview/coin-buttons.tsx

```diff
@@ -363,132 +357,99 @@ const CoinButtons = ({
   return (
     <Box
       display={Display.Flex}
-      justifyContent={JustifyContent.spaceEvenly}
+      justifyContent={JustifyContent.spaceBetween}
       width={BlockSize.Full}
+      gap={4}
     >
       {
         ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
-        <IconButton
-          className={`${classPrefix}-overview__button`}
-          iconButtonClassName={iconButtonClassName}
-          Icon={
-            <Icon
-              name={IconName.PlusAndMinus}
-              color={IconColor.iconDefault}
-              size={IconSize.Sm}
-            />
-          }
-          disabled={!isBuyableChain}
-          data-testid={`${classPrefix}-overview-buy`}
-          label={t('buyAndSell')}
-          onClick={handleBuyAndSellOnClick}
-          tooltipRender={(contents: React.ReactElement) =>
-            generateTooltip('buyButton', contents)
-          }
-        />
+        generateTooltip(
+          'buyButton',
+          <IconButton
+            className={`${classPrefix}-overview__button`}
+            Icon={
+              <Icon
+                name={IconName.PlusAndMinus}
+                color={IconColor.iconDefault}
+                size={IconSize.Sm}
+              />
+            }
+            disabled={!isBuyableChain}
+            data-testid={`${classPrefix}-overview-buy`}
+            label={t('buyAndSell')}
+            onClick={handleBuyAndSellOnClick}
+            width={BlockSize.Full}
+          />,
+        )
```

Comment:
Refactoring `IconButton` to no longer need `tooltipRender` as we only use it in coin buttons